documentation/adoc/cluster-controller.adoc

```diff
@@ -329,6 +329,55 @@ The resources created by the cluster controller into the Kubernetes/OpenShift cl
 * [connect-cluster-name]-connect Deployment which is in charge to create the Kafka Connect worker node pods
 * [connect-cluster-name]-connect Service which exposes the REST interface for managing the Kafka Connect cluster
 
+=== Provisioning Role-Based Access Control (RBAC) for the controller
+
+For the controller to function it needs permission within the Kubernetes/OpenShift cluster to interact with
```

Comment:
plural here ? "permissions" ... because next sentence you start with "These permissions ..."